libs/application/template-api-modules/src/lib/modules/templates/party-application/party-application.service.ts

```diff
@@ -34,6 +41,10 @@ type EndorsementListResponse =
     }
   | ErrorResponse
 
+export type PartyApplicationServiceOptions = {
```

Comment:
An interface would be better to describe this type. In general `interface` have better perfs than `type`

Reply:
Fixed